### DISCUSSION TITLE
token expired or invalid: 403

### DISCUSSION THREAD

**Discussion by AliNazarzade** [community/community#116920] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello, my account is free, but I can't use it, I get the following error that I posted to you, I logged out of the account in vscode and logged in again, and I tried my best to solve this problem, but it was useless.

token expired or invalid 403

**Comment by User132New** [comment]
Try generating new access token from your GitHub account.
Visit Docs for more details about how!
